tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -36,6 +36,12 @@
     "force_cleanup",
     default=False,
     help="Force resource cleanup, even if not created by this test run")
+# TODO(yashkt): We will no longer need this flag once Core exposes local certs
+# from channelz
+_CHECK_LOCAL_CERTS = flags.DEFINE_bool(
+    "check_local_certs",
+    default=False,
```

Comment:
@yashykt thank you for the change! One note: I've meant the option to disable check to be off by default (double negative). Something like `disable_check_local_certs` Sorry for the confusion. 
I think `check_local_certs` is better! But we need to make it on by default, so languages with cert support need no change, and languages without can simply add `--nocheck_local_certs`.
```suggestion
    default=True,
```

Reply:
@gnossen Please note